Tasks/AppCenterDistributeV1/task.json

```diff
@@ -12,10 +12,10 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 138,
+        "Minor": 139,
         "Patch": 0
     },
-    "releaseNotes": "Provide build id to distribution API for additional context.",
+    "releaseNotes": "Updating request source for AppCenter triggered builds.",
```

Comment:
This description doesn't make any sense and our App Center has a space.

Reply:
Hi @pniko , 
the release notes here provides the context of what got updated in this version of the task.

I agree, the updated release notes is not very relevant to the customer using.
Probably i can update to generic one `Added telemetry for additional context`. What do u say.

Reply:
@pniko updated. please check